admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -511,6 +511,12 @@ a look at the logs for further details."
 When the configuration test reports success, save your settings and check if the 
 users and groups are fetched correctly on the Users page.
 
+Syncing Users from the Command-Line
+-----------------------------------
+
+You can also sync users from an LDAP backend with your ownCloud database from the command-line, using the occ command. 
```

Comment:
You NEED sync users from an LDAP backend to your ownCloud database from the command-line, using the occ command. 
the occ user sync not only add the users for the first time, it adds all the search possibilities for each user too. That helps to find them without checking the LDAP.

It's really necessary to do it periodically over a cron job. for any update of users, quotas and search options.